invenio_jsonschemas/views.py

```diff
@@ -22,65 +22,25 @@
 # waive the privileges and immunities granted to it by virtue of its status
 # as an Intergovernmental Organization or submit itself to any jurisdiction.
 
-"""Views provided by Invenio-JSONSchemas."""
+"""Invenio module for building and serving JSONSchemas."""
 
-import datetime
-import time
+from __future__ import absolute_import, print_function
 
-from wsgiref.handlers import format_date_time
-
-from flask import Blueprint, jsonify, render_template
-
-from flask_babel import lazy_gettext as _
-
-from flask_breadcrumbs import default_breadcrumb_root, register_breadcrumb
-
-try:
-    from invenio.ext.cache import cache
-except ImportError:
-    from flask_cache import Cache
-    cache = Cache()
-
-from .api import get_schema_data, get_schemas, internal_schema_url
-
-from .utils import split_path, tree_insert, tree_sort
+from flask import Blueprint, abort, send_from_directory
 
+from .api import get_schema_dir
+from .errors import JSONSchemaNotFound
 
 blueprint = Blueprint(
-    'jsonschemas',
-    __name__,
-    url_prefix='/jsonschemas',
-    static_folder='static',
-    template_folder='templates',
+    'invenio_jsonschemas',
+    __name__
```

Comment:
Yes it is missing, I will do it in `app.register_blueprint` so that it can be overridden with a config variable.